views/term_table.erb

```diff
@@ -209,6 +209,29 @@
 
         </div>
     </div>
+      <div class="page-section page-section--code">
+        <div class="container">
+          <p>
+            Ruby programmer? It’s easy to use this data in your own app! Use the
+            <a href="https://medium.com/@everypolitician/introducing-the-everypolitician-gem-ab39db5b7e49">gem</a>
+            <!-- https://github.com/everypolitician/everypolitician-ruby -->
+            like this:
+          </p>
+          <pre>
+require 'everypolitician'
+country = Everypolitician::Index.new.country('<%= @page.country.slug %>')
+house = country.legislature('<%= @page.house.slug %>')
+house.popolo.terms.find_by(id: '<%= @page.current_term.id %>').memberships.each do |m|
+  puts "#{m.person.name} (#{m.party.name}): #{m.person.gender}"
+end
+          </pre>
```

Comment:
It's probably best to put this `</pre>` tag _immediately_ after the `end` on the line above. This should prevent the extra line break that's currently appearing when its rendered.